packages/web3-eth-accounts/test/fixtures/account.ts

```diff
@@ -29,3 +31,186 @@ export const invalidPrivateKeytoAccountData: [any, string][] = [
+		[
+			'0x67f476289210e3bef3c1c75e4de993ff0a00663df00def84e73aa7411eac18a6',
+			'123',
+			{
+				n: 8192,
+				iv: Buffer.from('bfb43120ae00e9de110f8325143a2709', 'hex'),
+				salt: Buffer.from(
+					'210d0ec956787d865358ac45716e6dd42e68d48e346d795746509523aeb477dd',
+					'hex',
+				),
+			},
+		],
+		{
+			version: 3,
+			address: 'cda9a91875fc35c8ac1320e098e584495d66e47c',
+			crypto: {
+				ciphertext: 'cb3e13e3281ff3861a3f0257fad4c9a51b0eb046f9c7821825c46b210f040b8f',
+				cipherparams: { iv: 'bfb43120ae00e9de110f8325143a2709' },
+				cipher: 'aes-128-ctr',
+				kdf: 'scrypt',
+				kdfparams: {
+					n: 8192,
+					r: 8,
+					p: 1,
+					dklen: 32,
+					salt: '210d0ec956787d865358ac45716e6dd42e68d48e346d795746509523aeb477dd',
+				},
+			},
+		},
+	],
+	[
+		[
+			'0x67f476289210e3bef3c1c75e4de993ff0a00663df00def84e73aa7411eac18a6',
+			'123',
+			{
+				n: 8192,
+				iv: 'bfb43120ae00e9de110f8325143a2709',
+				salt: '210d0ec956787d865358ac45716e6dd42e68d48e346d795746509523aeb477dd',
+			},
+		],
+		{
+			version: 3,
+			address: 'cda9a91875fc35c8ac1320e098e584495d66e47c',
+			crypto: {
+				ciphertext: 'cb3e13e3281ff3861a3f0257fad4c9a51b0eb046f9c7821825c46b210f040b8f',
+				cipherparams: { iv: 'bfb43120ae00e9de110f8325143a2709' },
+				cipher: 'aes-128-ctr',
+				kdf: 'scrypt',
+				kdfparams: {
+					n: 8192,
+					r: 8,
+					p: 1,
+					dklen: 32,
+					salt: '210d0ec956787d865358ac45716e6dd42e68d48e346d795746509523aeb477dd',
+				},
+			},
+		},
+	],
+	[
+		[
+			'0x348ce564d427a3311b6536bbcff9390d69395b06ed6c486954e971d960fe8709',
+			'123',
+			{
+				iv: 'bfb43120ae00e9de110f8325143a2709',
+				salt: '210d0ec956787d865358ac45716e6dd42e68d48e346d795746509523aeb477dd',
+				c: 262144,
+				kdf: 'pbkdf2',
+			},
+		],
+		{
+			version: 3,
+			id: 'e6a68d89-b03c-4e7d-82e0-079eebaa0da7',
+			address: 'b8ce9ab6943e0eced004cde8e3bbed6568b2fa01',
+			crypto: {
+				ciphertext: '76512156a34105fa6473ad040c666ae7b917d14c06543accc0d2dc28e6073b12',
+				cipherparams: { iv: 'bfb43120ae00e9de110f8325143a2709' },
+				cipher: 'aes-128-ctr',
+				kdf: 'pbkdf2',
+				kdfparams: {
+					dklen: 32,
+					salt: '210d0ec956787d865358ac45716e6dd42e68d48e346d795746509523aeb477dd',
+					c: 262144,
+					prf: 'hmac-sha256',
+				},
+				mac: '46eb4884e82dc43b5aa415faba53cc653b7038e9d61cc32fd643cf8c396189b7',
+			},
+		},
+	],
+];
+
+export const invalidEncryptData: [[any, string, any], Error][] = [
+	[
+		['0x67f476289210e3bef3c1c75e4de993ff0a00663df00def84e73aa7411eac18a', '123', {}],
+		new PrivateKeyLengthError(
+			'67f476289210e3bef3c1c75e4de993ff0a00663df00def84e73aa7411eac18a',
+		),
+	],
+	[
+		[
+			'0x348ce564d427a3311b6536bbcff9390d69395b06ed6c486954e971d960fe8709',
+			'123',
+			{
+				iv: 'bfb43120ae00e9de110f8325143a2709',
+				salt: '210d0ec956787d865358ac45716e6dd42e68d48e346d795746509523aeb477dd',
+				kdf: 'hkdf',
+			},
+		],
+		new InvalidKdfError(),
+	],
+];
+
+export const validDecryptData: [[any, string, CipherOptions]][] = [
```

Comment:
`validDecryptData: [[any, string, CipherOptions]][]`

should be

`validDecryptData: [[Keystore | string, string, CipherOptions]][]`

Also should have test cases for [nonStrict](https://github.com/ChainSafe/web3.js/blob/265c0fd4ef9941c07e6e00c64e80a516b4b552dc/packages/web3-eth-accounts/src/account.ts#L235)